addons/website/static/src/js/editor/widget_link.js

```diff
@@ -121,3 +122,14 @@ patch(LinkTools.prototype, {
         super._onPickSelectOption(...arguments);
     },
 });
+
+patch(Wysiwyg.prototype, {
```

Comment:
Actually, I just tested, the bug is also there in 16.0, you should retarget this PR. But actually the bug you solved with https://github.com/odoo/odoo/commit/dcf0880f836423ee82f3d283ffbf940a804bc13b in 17.0 was also there in 16.0... you will have to backport it too. Link me the PR once this is done please :pray:

Reply:
@qsm-odoo Now that @Mtaylorr has left Odoo, I wanted to do the backport for you, but when I tried to reproduce the issue on a 16.0+e runbot, I was not able to. The link is created correctly all the time, even though I do click on the dropdown to select the page (I tested "/contactus" and "/"). Am I missing something that is needed to reproduce the bug? Maybe that's related to the fact that I'm on macOS and the native dropdown reacts differently?

Reply:
Probably, macOS has been known to have issues like that (in this case, fixing an issue by chance but...). Don't worry I'll backport the other commit then I'll look to backport this one.

Reply:
https://github.com/odoo/odoo/pull/192081